src/server/search/search_family.cc

```diff
@@ -469,6 +469,53 @@ void SearchFamily::FtCreate(CmdArgList args, ConnectionContext* cntx) {
   cntx->SendOk();
 }
 
+void SearchFamily::FtAlter(CmdArgList args, ConnectionContext* cntx) {
+  CmdArgParser parser{args};
+  string_view idx_name = parser.Next();
+  parser.ExpectTag("SCHEMA");
+
+  // First, extract existing index info
+  optional<DocIndex> index_info;
+  cntx->transaction->Execute(
+      [idx_name, &index_info](auto* tx, EngineShard* es) {
+        if (es->shard_id() > 0)
+          return OpStatus::OK;
+
+        if (auto* idx = es->search_indices()->GetIndex(idx_name); idx != nullptr)
+          index_info = idx->GetInfo().base_index;
+        return OpStatus::OK;
+      },
+      false);
+
+  if (!index_info) {
+    cntx->transaction->Conclude();
+    return cntx->SendError("Index not found");
+  }
+
+  // Parse additional schema
+  optional<search::Schema> new_fields = ParseSchemaOrReply(index_info->type, parser, cntx);
```

Comment:
it is not possible to remove fields?

Reply:
I was surprised, but no, it only has SCHEMA ADD 🤷🏻‍♂️